src/middlewared/middlewared/plugins/crypto_/cert_profiles.py

```diff
@@ -105,15 +105,3 @@ async def profiles(self):
         configurations which can be used for creating certificates.
         """
         return CERTIFICATE_PROFILES
-
-    @accepts(roles=['CERTIFICATE_READ'])
-    @returns(Dict(
-        *[Dict(profile, additional_attrs=True) for profile in CSR_PROFILES],
-        example=CSR_PROFILES,
-    ))
-    async def certificate_signing_requests_profiles(self):
```

Comment:
UI should be using this for CSR profiles, seems like they are using the wrong endpoint atm

Reply:
@undsoft it is strange that the UI does not use this

Reply:
We call `webui.crypto.certificate_profiles`.

Reply:
That is not accurate and UI should be using this endpoint instead, i'll make a ticket

Reply:
Deja vu :) https://ixsystems.atlassian.net/browse/NAS-126028

Reply:
We missed specifying CSR profiles in that ticket, created a ticket for that https://ixsystems.atlassian.net/browse/NAS-131460

Reply:
Please rename that function and also put it under webui namespace. webui.crypto.csr_profiles